src/main/java/brobot/UI.java

```diff
@@ -0,0 +1,131 @@
+package brobot;
```

Comment:
I like how you packaged the classes with similar functionalities into sub-packages!